src/main/java/trackbot/task/Event.java

```diff
@@ -0,0 +1,44 @@
+package trackbot.task;
+
+import trackbot.ui.Parser;
+
+/**
+ * Event task with a start and end time.
+ */
+public class Event extends Task {
+    private String from;
```

Comment:
Perhaps a more descriptive name for these fields might be nice such as `startDate` and `endDate`.